### DISCUSSION TITLE
Nested projects at root

### DISCUSSION THREAD

**Discussion by SlipperyBrick** [spencerwmiles/vscode-task-buttons#105] [Ideas]
Hey guys, great extension! I've found a lot of value from it and use it on a daily basis. 

I wanted to know if the extension can read from nested `.vscode` folders that each have their own `tasks.json` and `settings.json` files?

![image](https://github.com/spencerwmiles/vscode-task-buttons/assets/36016443/ef126998-b545-49c0-9eb3-e4e56c884579)

My root has a `.vscode` folder with relevant json files, sub-project 1 and sub-project 2 also have their own `.vscode` folders too. I often find nested sub-projects within a root folder helps organize the overarching project and its code base.

I have tested trying to add buttons from sub-project 1 and 2 but noticed that the buttons don't show up. You have to have a `.vscode` folder with your `tasks.json` and a `settings.json` defined at root for the task buttons to show. Would be great if you supported project structures like this because tasks can stay contextual to the project their are meant for, rather than having potentially large json files full of tasks just to add task buttons.

**Comment by spencerwmiles** [comment]
Hey @SlipperyBrick sorry for taking so long to get back to you.

If you haven't already, you might want to try setting up a "workspace" https://code.visualstudio.com/docs/editing/workspaces/workspaces for this project. I think you might find that it will enable some, if not most of what you are looking for/are after using Task Buttons in your project.

Let me know if that helps. I've made it a mission this weekend to bring this extension some much needed love.

If that doesn't do it - please create an issue and I will make it a priority.
